scripts/us_usda/quickstats/process.py

```diff
@@ -0,0 +1,172 @@
+import requests
```

Comment:
Pls add the standard copyright template used across the repo.

Reply:
Done.